bin/winGet-latest.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+. ./bin/import-shared
+
+do_update () {
+    echo "$@ Changed..updating.."
+    for i in $SERVICES
+    do
+    ./bin/winSrvc.sh stop $i
+    done
+    git pull
+    ./bin/npm_install.sh
+    for i in $SERVICES
+    do
+    ./bin/winSrvc.sh start $i
```

Comment:
quality note: it is better to tab the code inside the for loops since it makes it easier to read